### PR TITLE
Improved JSON API. Resolves #1884.

### LINKED ISSUE #1884: Tweak To JSON API
Right now, the players json api (http://play.totalfreedom.me:28966/players) gives us the arrays "developers", "players", "telnetadmins", "superadmins", and "senioradmins". This is definitely helpful if you want to make an application that hooks into TFM, but you lack the ability to get a list of online admins.

For example, the TF website just looks for the list of admins; in this case, that's perfectly fine. However, if I want to create command permissions in an external plugin, I'd need to know what players are online, and which are admins. I can check to see if players online are listed as admins, but I can't tell if any of them are imposters.

I suggest that you add the array "onlineadmins" so that external programs can tell what players online are true admins. It shouldn't take but a few minutes to implement.

### PR DESCRIPTION
**Pull request by Hockeyfan360**
This commit allows external apps/plugins to see which admins are online through the HTTP API. This could be used to develop plugins quicker, without needing to integrate them directly into TFM.

It adds an 'onlineplayers' array to the json response for the players module.

I have tested this.

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/httpd/module/Module_players.java
Comment (ItzLevvie) [inline src/main/java/me/totalfreedom/totalfreedommod/httpd/module/Module_players.java]: There needs to be a space after ```if```

**Comment by Wild1145** [comment]
Is there any reason you couldn't just use the other online admin options?

**Comment by Hockeyfan360** [comment]
@Wild1145 The online players section shows everyone, including impostors. The new 'onlineadmins' array doesn't show impostors.

This allows external plugins to grant permissions to admins, for example. Now, those permissions won't be granted to impostors.

**Comment by Wild1145** [comment]
@Hockeyfan360 Okay, I am with you now. I was looking at the super, telnet and senior admins sections. But realised that they are for added admins, not online admins.

**Review by Wild1145 — APPROVED** [review]
Looks good to me.

**Review by Wild1145 — DISMISSED** [review]
Please resolve the merge conflicts on the base branch before this can be merged.

**Comment by JeromSar** [comment]
@Wild1145 Keep in mind that PRs are sometimes manually merged into another development branch.

**Comment by Wild1145** [comment]
@JeromSar Whatever branch its being merged into should be selected for the PR, it breaks the review process generally otherwise, and would break any CI system we implement in the future.

**Comment by JeromSar** [comment]
Will need a review from either @Wild1145 or @OxLemonxO.

**Review by Wild1145 — COMMENTED** [review]
Looks good - Just need to pull it down and test.

**Comment by JeromSar** [comment]
Correction: needs an *approved* review.

**Comment by Hockeyfan360** [comment]
@JeromSar It's been almost 2 1/2 months since I opened this PR. I've tested it, and it works fine. If the code looks alright to you guys, I'd appreciate if it were just merged. This PR would greatly simplify the process of making a plugin compatible with TFM.

In fact, I will provide a working copy of TFM with this addition if you request it. I just want to get this update pushed to the server as quickly as possible.

**Comment by mayokek** [comment]
I can confirm it's working.

**Comment by JeromSar** [comment]
@Hockeyfan360 I understand, but PRs take some time to review. As set by the guidelines, a PR should have at least 1 approved review. I would prefer it if either @Wild1145 or @OxLemonxO could look at this PR and test it out.

**Review by lemonsked — APPROVED** [review]
Looks good.

**Comment by Wild1145** [comment]
@OxLemonxO Have you tested this? Just trying to establish if it is ready to merge, if you havent when I get my IDE working I will do some testing and merge this into the TFM 1.12 Alpha branches.

**Comment by lemonsked** [comment]
@Wild1145 I haven't tested.

**Review by Wild1145 — APPROVED** [review]
Just got around to testing. Happy and merging into the 1.12 Alpha branch. Will then look to merge the entire 1.12 release into the devel branch in due course.

**Comment by Hockeyfan360** [comment]
:) im a contributor now, yay. Thanks for getting this done.

**Pull request merged**
